src/app/modules/truenas-connect/components/truenas-connect-spinner/truenas-connect-spinner.component.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { Spectator, createComponentFactory } from '@ngneat/spectator/jest';
+import { TruenasConnectSpinnerComponent } from './truenas-connect-spinner.component';
+
+describe('TruenasConnectSpinnerComponent', () => {
+  let spectator: Spectator<TruenasConnectSpinnerComponent>;
+
+  const createComponent = createComponentFactory({
+    component: TruenasConnectSpinnerComponent,
+  });
+
+  beforeEach(() => {
+    // Mock the animation method to prevent infinite loop
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    jest.spyOn(TruenasConnectSpinnerComponent.prototype as any, 'startAnimation').mockImplementation(() => {
+      // Do nothing - prevents the infinite animation loop
+    });
+
+    spectator = createComponent();
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it('should create spinner component', () => {
+    expect(spectator.component).toBeTruthy();
+  });
+
+  it('should render SVG with animation paths', () => {
+    const svg = spectator.query('svg');
+    expect(svg).toBeTruthy();
+
+    const paths = spectator.queryAll('path.exploded');
+    expect(paths).toHaveLength(5);
+  });
+
+  it('should call startAnimation on init', () => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const startAnimationSpy = jest.spyOn(spectator.component as any, 'startAnimation');
+    spectator.component.ngAfterViewInit();
```

Comment:
Should not call component methods directly in tests. We wanna test user actions. This is a life cycle hook so the code in this method should be automatically executed when the component loads.